Jenkinsfile.nightly

```diff
@@ -0,0 +1,23 @@
+pipeline {
+	agent { node { label 'node-08' } }
```

Comment:
We do not have any node matching `node-08` at the moment.
If we add more nodes/labels can we maybe chose more explicit names?

Reply:
Definitely in favour of that. `dependency-check`? Or should I reuse an existing node?

Reply:
`dependency-check` sounds good